.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-76c65b13280cd5782ace8050df45564ef17891f9
+a0b1693996f408c112b7a628860b7a754aaa4f77
```

Comment:
This is the wrong commit hash. Please use the following.
96c96ce89d2731846b733054f3cef8915f31a609